browser-test/src/end_to_end_enumerators.test.ts

```diff
@@ -0,0 +1,135 @@
+import { startSession, loginAsAdmin, loginAsGuest, AdminQuestions, AdminPrograms, ApplicantQuestions, selectApplicantLanguage, endSession } from './support'
+
+describe('End to end enumerator test', () => {
+  const programName = 'ete enumerator program';
+
+  it('Create nested enumerator and repeated questions', async () => {
+    const { browser, page } = await startSession();
+    page.setDefaultTimeout(4000);
+
+    await loginAsAdmin(page);
+    const adminQuestions = new AdminQuestions(page);
+    const adminPrograms = new AdminPrograms(page);
+
+    await adminQuestions.addNameQuestion('enumerator-ete-name');

```

Comment:
Nit. Building off of this string would probably be more readable.

```
const enumeratorBase = 'enumerator-ete';
const enumerator = enumeratorBase + '-enumerator';
const nestedEnumerator = enumerator + '-enumerator';
...etc
```

Reply:
Since the strings are so important in browser tests, I like keeping them as raw strings. Then there's no logic involved in trying to figure out what the expected strings should be, or how the questions are linked to their enumerators.